core/controller/src/main/scala/whisk/core/loadBalancer/LoadBalancerService.scala

```diff
@@ -185,23 +171,21 @@ class LoadBalancerService(config: WhiskConfig)(implicit val actorSystem: ActorSy
      * INVOKER MANAGEMENT
      */
 
+    /** Gets a producer which can publish messages to the kafka bus. */
+    private val messageProducer = new KafkaProducerConnector(config.kafkaHost, executionContext)
+    private val ackConsumer = new KafkaConsumerConnector(config.kafkaHost, "completions", "completed")
     private val pingConsumer = new KafkaConsumerConnector(config.kafkaHost, "health", "health")
-    pingConsumer.onMessage((topic, _, _, bytes) => {
-        val raw = new String(bytes, "utf-8")
-        PingMessage.parse(raw) match {
-            case Success(p: PingMessage) => invokerPool ! p
-            case Failure(t)              => logging.error(this, s"failed processing message: $raw with $t")
-        }
-    })
-
     private val consul = new ConsulClient(config.consulServer)
     private val invokerFactory = (f: ActorRefFactory, name: String) => f.actorOf(InvokerActor.props, name)
-    private val invokerPool = actorSystem.actorOf(InvokerPool.props(invokerFactory, consul.kv, invoker => {
-        clearInvokerState(invoker)
-        logging.info(this, s"cleared loadbalancer state of $invoker")(TransactionId.invokerHealth)
-    }))
+    // Do not create the invokerpool, if it is not possible to create the test action to recover the invokers.
+    private val invokerPool = InvokerPool.createTestActionForInvokerHealth(entityStore).map { _ =>
```

Comment:
should this construction be more deliberate and explicit here and just fail via a `require` or `assert`?